src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -273,7 +274,7 @@
 					IsEnabled="{x:Bind ViewModel.Commands.NavigateBack.IsExecutable, Mode=OneWay}"
 					Style="{StaticResource AddressToolbarButtonStyle}"
 					ToolTipService.ToolTip="{x:Bind ViewModel.Commands.NavigateBack.LabelWithHotKey, Mode=OneWay}">
-					<FontIcon FontSize="14" Glyph="{x:Bind ViewModel.Commands.NavigateBack.Glyph.BaseGlyph, Mode=OneWay}" />
+					<s:ArrowGlyph Arrow="Back" FontSize="14" />
```

Comment:
What's happening here?

Reply:
Rewinds the direction of the arrows for the RTL culture, this is necessary for correction functions. Simply setting the flow direction in xaml is not enough.